test/extended/operators/olm.go

```diff
@@ -168,6 +168,36 @@ var _ = g.Describe("[sig-operator] an end user can use OLM", func() {
 	g.It("can subscribe to the operator", func() {
 		g.By("Cluster-admin user subscribe the operator resource")
 
+		// skip test if redhat-operators is not present or disabled
+		spec, err := oc.AsAdmin().Run("get").Args("operatorhub/cluster", "-o", "jsonpath='{.spec}'").Output()
```

Comment:
This really should be in a function

Reply:
Reworked into `hasRedHatOperatorsSource`